packages/blockchain-wallet-v4-frontend/src/modals/Settings/UpgradeNow/template.success.tsx

```diff
@@ -0,0 +1,249 @@
+  const isUserTierZero = userCurrentTier === TIER_TYPES.NONE
+  const isUserTierSilver =
+    currentTier === TIER_TYPES.SILVER || currentTier === TIER_TYPES.SILVER_PLUS
+
+  const isGoldInreview = goldTier.state === 'under_review' || goldTier.state === 'pending'
+
+  return (
+    <Wrapper>
+      <HeaderWrapper>
+        <IconsContainer>
+          <Link onClick={props.handleClose} data-e2e='upgradeNowCloseButton'>
+            <Image width='26px' name='arrow-left' />
+          </Link>
+        </IconsContainer>
+        <Title color='textBlack' size='24px' weight={600} style={{ marginTop: '18px' }}>
+          <FormattedMessage id='scenes.interest.verifyid' defaultMessage='Upgrade Now' />
+        </Title>
+        <Text
+          color='textBlack'
+          size='16px'
+          weight={500}
+          style={{ fontStyle: 'normal', marginTop: '8px' }}
+        >
+          <FormattedMessage
+            id='modals.upgradenow.subtitle'
+            defaultMessage='Start buying & selling crypto today.'
+          />
+        </Text>
+      </HeaderWrapper>
+
+      <MainContent>
+        {(isUserTierSilver || isUserTierZero) && (
+          <UpgradeContainer>
+            <Image name='blockchain-gold' size='20px' />
+            <TierTitle>
+              <FormattedMessage
+                id='modals.tradinglimits.upgrade.gold_limits'
+                defaultMessage='Gold Limits'
+              />
+            </TierTitle>
+            <UpgradeSubtitle>
+              <FormattedMessage
+                id='modals.tradinglimits.upgrade.tier.silver.description'
+                defaultMessage='Verify your ID and link your bank to Buy & Sell Crypto straight from your wallet.'
+              />
+            </UpgradeSubtitle>
+            <UpgradeDescription>
+              <FormattedMessage
+                id='modals.upgradenow.gold_details'
+                defaultMessage='Connect your bank or card to your Wallet. Hold cash in your wallet. Earn crypto with Rewards.'
+              />
+              {` `}
+              <span>
+                <FormattedMessage
+                  id='modals.upgradenow.gold_details_additional'
+                  defaultMessage='Includes all Silver limits and features.'
+                />
+              </span>
+            </UpgradeDescription>
+            {interestEDDStatus?.eddSubmitted || isGoldInreview ? (
+              <StatusCartridge>
+                <BlueCartridge fontSize='12px'>
+                  <FormattedMessage
+                    id='modals.upgradenow.in_manual_review'
+                    defaultMessage='In Manual Review'
+                  />
+                </BlueCartridge>
+              </StatusCartridge>
+            ) : (
+              <Button
+                fullwidth
+                size='16px'
+                height='48px'
+                nature='light'
+                data-e2e='upgradeNowUnlockGoldLimits'
+                type='button'
+                onClick={() =>
```

Comment:
not a huge deal, but you could wrap this in a `useCallback()` hook instead of anonymous function